sql/moz-fx-data-marketing-prod/ga_derived/www_site_events_metrics_v2/schema.yaml

```diff
@@ -82,11 +82,11 @@ fields:
 - mode: NULLABLE
   name: total_events
   type: INT64
-  description: Total Events - The total number of events reported
+  description: Total Events - The total number of non-page view events belonging to sessions with 1 or more engaged_session_event
```

Comment:
For info, which filter that limits this total to  of non-page view events?

Reply:
the visits = 1 filter (visits in the upstream table is defined as a session with 1 or more "engaged_session_event")